repositories/user_repo.py

```
@@ -52,6 +52,20 @@ async def update_user_unlocked_chapters(userId,chapterId):
+
+
+async def update_user_subscription(userId: str, subscription: dict):
+    try:
+        user = await db.users.find_one({"_id": ObjectId(userId)})
+        if not user:
+            return None
+        db.users.update_one(
+            filter={"_id": ObjectId(userId)},
+            update={"$set": {"subscription": subscription}},
+        )
+        return True
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)